### DISCUSSION TITLE
[EDA] 살짝 비스듬한 데이터 처리하기

### DISCUSSION THREAD

**Discussion by Sooho-Kim** [bcaitech1/p4-fr-soccer#18] [Experiment]
* 참고 : https://ichi.pro/ko/opencvleul-sayonghayeo-tegseuteu-imijileul-jadong-eulo-giulgi-bojeong-jigseonhwa-haneun-bangbeob-176759562404831 (동빈님 제공)

* 샘플(가로)
    * 잘 적용된 경우의 샘플
    * ![image](https://user-images.githubusercontent.com/67626878/119682630-012fd200-be7e-11eb-8e7d-f4aaeb231882.png)
    * ![image](https://user-images.githubusercontent.com/67626878/119682844-2c1a2600-be7e-11eb-8020-17ca26e02fc9.png)
    * ![image](https://user-images.githubusercontent.com/67626878/119683077-5835a700-be7e-11eb-960e-de02c8d0581d.png)

    * 적용 안해도 괜찮은 경우의 샘플 
    * ![image](https://user-images.githubusercontent.com/67626878/119682671-0856e000-be7e-11eb-8dca-0470dffe0790.png)
    * ![image](https://user-images.githubusercontent.com/67626878/119682961-4227e680-be7e-11eb-9664-9f8cb718b605.png)

* 샘플(세로)
    * 자동으로 회전해주는 바람에 아래 그림처럼 잘리는 현상 발생 이미지의 출력 크기를 변경해줄 필요가 있어보입니다.
    * ![image](https://user-images.githubusercontent.com/67626878/119761563-f9584800-bee6-11eb-9737-aa9ac107ac41.png)

    * 잘된 경우
    * ![image](https://user-images.githubusercontent.com/67626878/119761828-800d2500-bee7-11eb-9794-c5d810f7be1f.png)

* 살짝 수정한 결과
    * ![image](https://user-images.githubusercontent.com/67626878/119767449-c8314500-bef1-11eb-8bdb-75ecbc4d89d4.png)

* 특이케이스(뒤집었을 때 0+1)
    * ![image](https://user-images.githubusercontent.com/67626878/119770140-c5851e80-bef6-11eb-854a-668659218a7b.png)

**Comment by bellamy-k** [comment]
데이터왕 수호님ㄷㄷ
